LIPs/LIP-poll-based-governance.md

```diff
@@ -0,0 +1,26 @@
+    LIP: <to be assigned>
+    Title: Poll Based Governance
+    Author: Yondon Fu (@yondonfu), Doug Petkanics (@dob)
+    Type: Meta
```

Comment:
Could you just describe why "Meta" is the right type of LIP here? Is it because this is an LIP about the LIP process itself?

Reply:
Yeah that was my thought and because these LIPs do not result in any binding on-chain action and the expectation to carry out a result of a poll is defined by the process as opposed to being enforced by code.